HARDWARE.md

```diff
@@ -0,0 +1,13 @@
+# Bottlerocket test hardware
```

Comment:
"test hardware" doesn't sound correct to me. Also I think the file should be called `SUPPORTED-HARDWARE.md`.

Seems like we want two tiers:
* official support - hardware that the project or partners have access to and qualify builds on prior to release
* community support - hardware that's been reported to work

We might also want to consider using the wiki feature for the other category, if we want to streamline the ability to mark new platforms for the "community support" level.

@stockholmux what do you think?

Reply:
@bcressey I'd like to avoid introduce anything new with the concept of 'community' involved. It creates an artificial line.

I would suggest 'confirmed support' and 'reported support'.

I also like markdown files in the repo because there is a consistent contribution and approval model. For example, if there was some small change that enabled XYZ hardware to work, one PR would handle the change and the update to the support. That and we _want_ folks to contribute even if it's just modifying some markdown.